### ISSUE TITLE
Why the samples are not as good as nv-waveglow?

### ISSUE THREAD

**Issue by megazone87** [b04901014/waveglow-tensorflow#4]

**Comment by b04901014** [comment]
I'm not sure what configuration you use to train both networks.
However, it's reasonable to get a lower quality if you are comparing to the official samples Nvidia provides.
They use 8 K80 to afford a much larger batch size for training.
SGD with small batch size can be viewed as adding additional noise to the gradient when training.
Such noise sometimes helps generalization but on the other side hinders the model from reaching the local minimum.
Therefore we may converge (get stuck) before reaching the max quality.

**Comment by b04901014** [comment]
Issue closed due to inactivity.
